stubs/wlroots/__init__.pyi

```diff
@@ -0,0 +1 @@
+from ._ffi import ffi as ffi, lib as lib
```

Comment:
I wonder if having this stub file here is throwing things off? I think we can get rid of this, if it is needed, it should be able to pull it from the actual package.